### PR TITLE
[redis] Import Bitnami Redis Chart `v13.0.1`

### PR DESCRIPTION
**Pull request by bastjan**
This PR imports `bitnami/redis` at version `v13.0.1` commit `7c48ded7b69ea8f3646edf9da5b4a7f5590bec1b` [link](https://github.com/bitnami/charts/tree/7c48ded7b69ea8f3646edf9da5b4a7f5590bec1b/bitnami/redis).

~~Please double check the changes to the `README.md`:~~ `README.md` is autogenerated 🤦‍♀️.

#### What this PR does / why we need it:

This PR is needed to further customise the Redis chart.

#### Checklist

- [x] [DCO](https://github.com/appuio/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Variables are documented in the values.yaml using the format required by [Helm-Docs](https://github.com/norwoodj/helm-docs#valuesyaml-metadata).
- [x] Title of the PR contains starts with chart name e.g. `[chart]`

### CODE REVIEW

**Review by simu — CHANGES_REQUESTED** [review]
We need to import the v13 version of the Bitnami Redis chart, as that's the one we currently use in production and there's no upgrade path from v13 to v14 (except creating a new redis instance and copying data over), cf. https://github.com/bitnami/charts/tree/master/bitnami/redis#to-1400

**Comment by bastjan** [comment]
> We need to import the v13 version of the Bitnami Redis chart 

Must have missed this 🙈 .Found the commit responsible for `v13.0.1` after some git pickaxe action.

File: appuio/redis/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+- keyvalue
+- database
+maintainers:
+  - name: APPUiO Team
+    email: info@appuio.ch
+name: redis
+sources:
+- https://github.com/bitnami/bitnami-docker-redis
+- http://redis.io/
+version: 13.0.1
```
Comment (simu) [inline appuio/redis/Chart.yaml:27]: We'll probably want to adjust the version, not sure what to though.

**Review by simu — APPROVED** [review]
LGTM overall, as noted inline, we may want to adjust the chart version. I'm not sure what the proper approach there is though.

File: appuio/redis/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+keywords:
+- redis
+- keyvalue
+- database
+maintainers:
+  - name: APPUiO Team
+    email: info@appuio.ch
+name: redis
+sources:
+- https://github.com/bitnami/bitnami-docker-redis
```
Comment (corvus-ch) [inline appuio/redis/Chart.yaml:25]:
```suggestion
- https://github.com/appuio/charts/tree/master/appuio/redis
```

File: appuio/redis/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+- name: common
+  repository: https://charts.bitnami.com/bitnami
+  tags:
+  - bitnami-common
+  version: 1.x.x
+description: Open source, advanced key-value store. It is often referred to as a data
+  structure server since keys can contain strings, hashes, lists, sets and sorted
+  sets.
+home: https://github.com/appuio/charts/tree/master/appuio/redis
+icon: https://bitnami.com/assets/stacks/redis/img/redis-stack-220x234.png
```
Comment (corvus-ch) [inline appuio/redis/Chart.yaml:15]: We might want to copy that icon to this repository and serve it from there.

**Review by corvus-ch — APPROVED** [review]
I have not review all the chart templates but only the chart definition. I trust that the correct version got imported.

File: appuio/redis/Chart.yaml (hunk 1 above)
Comment (bastjan) [reply appuio/redis/Chart.yaml:27]: INput from daily sync: Let's start at `1.0.0`

File: appuio/redis/Chart.yaml (hunk 2 above)
Comment (bastjan) [reply appuio/redis/Chart.yaml:25]: That's the link the the underlying `Dockerfile` source code.

**Pull request merged**